frontend/src/app/main/ui/ds/foundations/typography.mdx

```diff
@@ -0,0 +1,135 @@
+import { Canvas, Meta } from "@storybook/blocks";
+import * as TypographyStories from "./typography.stories";
+
+<Meta of={TypographyStories} />
+
+## Fonts
+
+Our preferred font is "Work Sans", it is used throughout our interface.  
+In case of having the interface in Farsi or Arabic this font will be replaced by "Vazirmatn".
+
+In the code area "Roboto Mono" will be used.
+
+## Variants
+### Title variants 
+
+#### Display (default)
+
+- Used in the majority of the interface, and **it is the default variant**.
+
+  font-weight: 400;
+
+  line-height: 1.2;
+
+  font-size: 36px;
+
```

Comment:
Instead of writing the spec like this, let's render an actual element (either the new `<Heading>` or `<Text>` when we have it):

```mdx
<Heading level="2" typography="display">Display 16px / 1.2 Weight 400</Heading>
```